packages/@expo/cli/src/lint/index.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+import chalk from 'chalk';
+
+import { Command } from '../../bin/cli';
+import { assertArgs, getProjectRoot, printHelp } from '../utils/args';
+
+export const expoLint: Command = async (argv) => {
+  const args = assertArgs(
+    {
+      // Types
+      '--help': Boolean,
+      // Aliases
+      '-h': '--help',
+    },
+    argv
+  );
+
+  if (args['--help']) {
+    printHelp(
+      `Utility for setting up linting on an Expo project`,
+      chalk`npx expo lint`,
+      [`-h, --help                 Usage info`].join('\n')
+    );
```

Comment:
Seems that chalk is not necessary here to add, as it isn't using any chalk interpolation features 😄 

```suggestion
    printHelp(
      'Utility for setting up linting on an Expo project',
      'npx expo lint',
      '-h, --help                 Usage info'
    );
```